packages/core/src/bodl/bodl-emitter-service.ts

```diff
@@ -45,16 +45,18 @@ export default class BodlEmitterService implements BodlService {
                 currency,
                 lineItems,
                 id,
-                coupons
+                coupons,
             },
+            channelId,
```

Comment:
Same as https://github.com/bigcommerce/checkout-sdk-js/pull/1670/files#r1013808067 comment, but for checkout interface.

Reply:
See you have already done this.

Reply:
yep, I added